config/go-app-ussd_higherhealth_healthcheck.afr_ZA.json

```diff
@@ -0,0 +1,262 @@
+{
+    "": {
+        "pot-creation-date": "2020-07-27 01:03:+0000",
+        "project-id-version": "",
+        "language": "sot",
+        "mime-version": "1.0",
+        "content-type": "text/plain; charset=utf-8",
+        "content-transfer-encoding": "8bit",
+        "po-revision-date": "2020-07-28 12:00+0200",
+        "last-translator": "",
+        "language-team": "",
+        "x-generator": "Poedit 2.4"
+    },
+    "Welcome back to the The National Department of Health's COVID-19 Service\n\nReply": [
+        null,
+        "Welkom terug by Die Nasionale Department van Gesondheid se COVID-19 Diens\n\nAntwoord"
+    ],
+    "Continue where I left off": [
+        null,
+        "Gaan voort waar ek opgehou het"
+    ],
+    "Start over": [
+        null,
+        "Begin oor"
+    ],
+    "Welcome to HealthCheck.\nPlease select your preferred language.\nReply\n": [
+        null,
+        ""
+    ],
+    "Afrikaans": [
+        null,
+        ""
+    ],
+    "English": [
+        null,
+        ""
+    ],
+    "Sotho": [
+        null,
+        ""
+    ],
+    "Xhosa": [
+        null,
+        ""
+    ],
+    "Zulu": [
+        null,
+        ""
+    ],
+    "The HIGHER HEALTH HealthCheck is your risk assessment tool. Help us by answering a few questions about you and your health.\n\nReply": [
+        null,
+        "Die Nasionale Departement van Gesondheid bedank jou vir jou bydra. Stop die verspreiding van COVID-19.\n\nAntwoord"
+    ],
+    "This service works best when you select numbers from the list": [
+        null,
+        "Hierdie diens werk die beste wanneer jy nommers van die lys af kies"
+    ],
+    "START": [
+        null,
+        "BEGIN"
+    ],
+    "Confirm that you're responsible for your medical care & treatment. This service only provides info.\n\nReply": [
+        null,
+        "Bevestig dat jy verantwoordelik is vir jou eie mediese sorg en behandeling. Hierdie diens verskaf slegs inligting.\n\nAntwoord"
+    ],
+    "Please use numbers from list. Confirm that u're responsible for ur medical care & treatment. This service only provides info.\n\nReply": [
+        null,
+        "Bevestig dat jy verantwoordelik is vir jou eie mediese sorg en behandeling. Hierdie diens verskaf slegs inligting.\n\nAntwoord"
+    ],
+    "YES": [
+        null,
+        "JA"
+    ],
+    "NO": [
+        null,
+        "NEE"
+    ],
+    "MORE INFO": [
+        null,
+        "MEER INLIGTING"
+    ],
+    "You can return to this service at any time. Remember, if you think you have COVID-19 STAY HOME, avoid contact with other people and self-quarantine.": [
+        null,
+        "Jy kan enige tyd na hierdie diens terug keer.\nOnthou, as jy dink dat jy COVID-19 het, BLY BY DIE HUIS, vermy kontak met ander mense en self-kwarantyn."
```

Comment:
The english doesn't have a newline. I think we should keep the formatting the same between languages.